client/src/components/misc_util_components.js

```diff
@@ -24,13 +23,32 @@ const ExternalLink = ({ children, href, title }) => (
 
 class Format extends React.PureComponent {
   render() {
-    const { type, content, style, className } = this.props;
+    const {
+      type,
+      content,
+      style,
+      className,
+      in_parenthesis,
+      prefix,
+    } = this.props;
+
+    const formatted_content = formats[type](content);
+
+    const prefixed_content = prefix
+      ? `<span>${prefix}${formatted_content}</span>`
+      : formatted_content;
+
+    const parenthesis_content = in_parenthesis
+      ? `<span>(${prefixed_content})</span>`
+      : prefixed_content;
 
     return (
       <span
         style={style}
         className={className}
-        dangerouslySetInnerHTML={{ __html: formats[type](content) }}
+        dangerouslySetInnerHTML={{
+          __html: parenthesis_content,
```

Comment:
Real minor nitpick, but at this point `parenthesis_content` would be more accurately named something like `potentially_parenthesised_potentially_prefixed_content`. 

It's a bit fiddly to name things well when making a series of nested conditional modifications. Could instead consider the prefix and parentheses to be part of the formatting, maybe even collapse the application of formatting in to a single chain and just have a formatting pipeline like
```
const formatted_content = _.chain(content)
  .thru(formats[type])
  .thru((content) => (prefix ? `<span>${prefix}${content}</span>` : content))
  .thru((content) => (in_parenthesis ? `<span>(${content})</span>` : content))
  .value();
```